examples/minnietwitter/src/main/java/amino/run/appexamples/minnietwitter/MinnieTwitterMain.java

```diff
@@ -71,75 +74,73 @@ private static void printStatistics() {}
      * (Sungwook Moon, 12/5/2017) This is a temporary method for demo to show shift policy. This
      * method adds a single user and send tweet messages for predefined times.
      */
-    private static void ExecuteSingleUserDemo(
-            InetSocketAddress hostAddr, InetSocketAddress omsAddr) {
+    private static void ExecuteSingleUserDemo(InetSocketAddress hostAddr, InetSocketAddress omsAddr)
```

Comment:
This method is not being used anymore. shall we remove it ?

Reply:
Yes, agreed.

Reply:
Done.